### PR TITLE
For sync qg

### PR DESCRIPTION
**Pull request by adkumar1**

### CODE REVIEW

File: charts/orchestrator/values.yaml
Comment (adityagajbhiye9) [inline charts/orchestrator/values.yaml:1]: suggestion: we should use tractusx image

Comment (adkumar1) [reply charts/orchestrator/values.yaml:1]: this is for our catenax-ng closed repo image

File: DOCKER_NOTICE.md
[the remark this answers is not in the document]
Comment (adkumar1) [reply DOCKER_NOTICE.md:1]: chnaged to 17.0.10_7-jdk

File: Dockerfile
[the remark this answers is not in the document]
Comment (adkumar1) [reply Dockerfile:1]: its already there in tractusX

**Pull request merged**